src/features/deeplinks/routing.test.ts

```diff
@@ -0,0 +1,78 @@
+import * as datesLib from 'libs/dates'
+
+import { DEEPLINK_TO_SCREEN_CONFIGURATION } from './routing'
+
+describe('DEEPLINK_TO_SCREEN_CONFIGURATION', () => {
+  describe('link mot-de-passe-perdu', () => {
+    it('should return Home page when no params are passed', () => {
+      const configureScreen = DEEPLINK_TO_SCREEN_CONFIGURATION['mot-de-passe-perdu']()
+      expect(configureScreen.screen).toBe('Home')
+    })
+
+    it('should return Home page when params are invalid', () => {
+      let configureScreen = DEEPLINK_TO_SCREEN_CONFIGURATION['mot-de-passe-perdu']({
+        tok: '',
+        expiration_timestamp: '',
+      })
+      expect(configureScreen.screen).toBe('Home')
+
+      configureScreen = DEEPLINK_TO_SCREEN_CONFIGURATION['mot-de-passe-perdu']({
+        tok: '',
+        expiration_time: '',
+      })
+      expect(configureScreen.screen).toBe('Home')
+    })
+
+    it('should return ResetPasswordExpiredLink page when expiration_timestamp is expired', () => {
+      jest.spyOn(datesLib, 'isTimestampExpired').mockReturnValue(true)
+      const params = { token: 'token', expiration_timestamp: '11111111' }
+      const configureScreen = DEEPLINK_TO_SCREEN_CONFIGURATION['mot-de-passe-perdu'](params)
+
+      expect(configureScreen.screen).toBe('ResetPasswordExpiredLink')
+    })
+
+    it('should return ReinitializePassword page when expiration_timestamp is expired', () => {
+      jest.spyOn(datesLib, 'isTimestampExpired').mockReturnValue(false)
+      const params = { token: 'token', expiration_timestamp: '11111' }
+      const configureScreen = DEEPLINK_TO_SCREEN_CONFIGURATION['mot-de-passe-perdu'](params)
+
+      expect(configureScreen.screen).toBe('ReinitializePassword')
+      expect(configureScreen.params).toEqual({ token: params.token, expiration_timestamp: 11111 })
+    })
+  })
+
+  describe('link default', () => {
```

Comment:
pareil pour ce test et les suivants, un it.each serait plus lisible.

Reply:
À voir dés que il y aura plus de logique "pré-navigation" ;)

Reply:
Ah, je sais pourquoi c'est pas possible : il y a de potentiels mock à faire seulement dans le cas d'une route et pas d'une autre.
Par exemple : `jest.spyOn(datesLib, 'isTimestampExpired').mockReturnValue(true)`